man/docker-images.1.md

```diff
@@ -103,6 +103,24 @@ which displayed different visualizations of the image data. Docker core removed
 this functionality in the 1.7 version. If you liked this functionality, you can
 still find it in the third-party dockviz tool: https://github.com/justone/dockviz.
 
+## Listing images in a desired format
+
+You can use the `--format` parameter to list only the image attributes required
+and in the order required. For example, if the image ID, Tag, and REPOSITORY 
+are required, in that order, then the following command is used:
+
+    docker images --format "{{.ID}}: {{.Tag}}\t{{.Repository}}"
```

Comment:
Perhaps use the `table` format here, so that it nicely aligns the column if needed;

```
docker images --format "table {{.ID}}\t{{.Tag}}\t{{.Repository}}"

IMAGE ID           TAG                              REPOSITORY
d119afad64c7       master                           docker-dev
e373a5ed31f3       move-logdriver-check-to-daemon   docker-dev
98d07712be25       <none>                           <none>
72a876f86a11       <none>                           <none>
6b70474fa7e9       latest                           docs
```